### ISSUE TITLE
aria-expanded=false is never applied to tree view elements

### ISSUE THREAD

**Issue by MRayermannMSFT** [Microsoft/AzureStorageExplorer#352]
Fixing this should close: https://devdiv.visualstudio.com/DevDiv/_workitems/edit/518173

Long story short, it looks like Knockout is only adding aria-expanded (with a value of true) to our tree nodes, not sure if this is a bug on Knockout's part but it should be easy to add a slightly more complicated if which forces the correct values to be used, and then no attribute to be added if the node is not expandable.

**Comment by MRayermannMSFT** [comment]
Merged into master for 1.3.0

**Comment by MRayermannMSFT** [comment]
Testers have closed the bug!
